packages/core/src/lib/passcode.test.ts

```diff
@@ -80,11 +80,11 @@ describe('createPasscode', () => {
 
   it('should disable existing passcode', async () => {
     const email = 'jony@example.com';
-    const sessionId = 'sessonId';
-    mockedFindUnconsumedPasscodesBySessionIdAndType.mockResolvedValue([
+    const jti = 'jti';
```

Comment:
What does jti stand for?

Reply:
https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.7

Reply:
Got it, thanks!